docs/sources/mimir/configure/configure-resource-utilization-based-ingester-read-path-limiting.md

```diff
@@ -0,0 +1,28 @@
+---
+aliases:
+  - ../operators-guide/configure/configure-resource-utilization-based-ingester-read-path-limiting/
+description: Learn how to configure Grafana Mimir for resource utilization based ingester read path limiting.
+menuTitle: Resource utilization based ingester read path limiting
+title: Configure resource utilization based ingester read path limiting
+weight: 120
+---
+
+# Configure resource utilization based ingester read path limiting
+
+As an **experimental** feature, Grafana Mimir allows you to configure limits for CPU and/or memory utilization which can cause the ingester to reject incoming read requests in order to protect the write path. The ingester's model of memory utilization
```

Comment:
[non blocking] Same comment about experimental here (see the other comment).

Reply:
Donie, thanks.